src/backend/marsha/core/templates/core/lti_development.html

```diff
@@ -229,7 +229,7 @@ <h2>Iframe LTI Resource setup</h2>
             <option value="videos" selected>video</option>
             <option value="documents">document</option>
             <option value="meetings">meeting</option>
-            <option value="markdown_documents">Markdown document</option>
```

Comment:
The same change must be done on line 334 of this file

Reply:
These changes breaks the development view : when we click on a `last_objects` the `data-model` corresponds to the `RESOURCE_NAME` (https://github.com/openfun/marsha/blob/ef7a10f37d2e1343216be9420656c9c05e89ab8e/src/backend/marsha/core/templates/core/lti_development.html#L162) but in this case the `RESOURCE_NAME` is `markdown_documents` so the corresponding option is not found. Not a big deal of course :) but I only hope this does not reflect a bigger problem.

Reply:
Good catch, I modified that.